WordPress/build.gradle

```diff
@@ -238,6 +245,24 @@ android {
     }
 }
 
+def addBuildConfigFieldsFromPrefixedProperties(variant, properties, prefix) {
```

Comment:
nit: unsure about the vocabulary, should that be called `variant` or `flavor` or something else?
Named it `variant` because apparently [the Gradle `BuildConfigField` class is part of the `variant` namespace](https://developer.android.com/reference/tools/gradle-api/4.1/com/android/build/api/variant/BuildConfigField) so I figured that meant it sense, but happy to be corrected.